build.sbt

```diff
@@ -31,82 +31,26 @@ lazy val recryptSettings = Seq(
 
   // Test
   libraryDependencies ++= Seq(
-    "org.scalatest" %%% "scalatest" % "3.0.9" % "test",
-    "org.scalacheck" %%% "scalacheck" % "1.15.0" % "test",
-    "org.typelevel" %%% "spire-laws" % "0.14.1" % "test",
-    "org.typelevel" %%% "discipline" % "0.10.0" % "test"
-  ) ++ Seq( // Core dependencies.
-    "org.typelevel" %%% "spire" % "0.15.0",
-    "org.scodec" %%% "scodec-bits" % "1.1.18",
-    "org.typelevel" %%% "cats-effect" % "2.1.4"
+    "org.scalatest" %% "scalatest" % "3.2.9" % "test",
+    "org.scalacheck" %% "scalacheck" % "1.15.4" % "test",
+    "org.typelevel" %% "spire-laws" % "0.17.0" % "test",
+    "org.typelevel" %% "discipline-scalatest" % "2.1.5" %"test",
+    ) ++ Seq( // Core dependencies.
+    "org.typelevel" %% "spire" % "0.17.0",
+    "org.scodec" %% "scodec-bits" % "1.1.27",
+    "org.typelevel" %% "cats-effect" % "2.5.1"
   ),
-  //List is from https://tpolecat.github.io/2017/04/25/scalac-flags.html
```

Comment:
None of this is necessary anymore?

Reply:
We get similar by adding the tpolecat's plugin
`addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.18") `